aiowebostv/webos_client.py

```diff
@@ -314,6 +282,7 @@ async def connect_handler(self, res: Future) -> None:
         self.callback_queues = {}
         self.callback_tasks = {}
         self.futures = {}
+        self.tv_info.clear()
```

Comment:
This is a change from previous implementation, the states is cleared upon connect and not upon disconnect so that in case a client was disconnected this data is still available to be used in diagnostics